lib/unleash/strategy/flexible_rollout.rb

```diff
@@ -32,6 +33,12 @@ def is_enabled?(params = {}, context = nil)
 
       private
 
+      def context_is_sufficient?(stickiness, context)
+        return true if ['random', 'default'].include?(stickiness)
```

Comment:
Note that it will not evaluate the strategy randomly when the stickiness is set to `random`. 

You sure that's the desired behaviour?

Maybe it's worth having a rspec test just for the`'stickiness' => 'radom'` case? (where a couple of different seeds are hard coded?)

Reply:
this will just let it continue to the resolve stickiness like it did before, this is just a safe guard. It returns to the is_enabled method, not to the caller.

Reply:
but what happens up the stack, in the `is_enabled?` method? it returns right away, no? or is the boolean logic a bit hard to read?

Reply:
it only returns false when its supposed to, if the context is sufficient it continues...